### PR TITLE
Adding option to select Infra nodes flavor

### LINKED ISSUE #64: Include a parameter to select the flavor used by infra nodes on rosa cli script
Based on this PR https://github.com/openshift/rosa/pull/241, adding same parameter to use it from the wrapper

### PR DESCRIPTION
**Pull request by morenod**
--rosa-flavor=<<AWS flavor>> will select AWS flavor for infra nodes.

No validations done of flavor exists or anything related

### CODE REVIEW

**Review by chentex — APPROVED** [review]
LGTM

File: rosa/rosa-wrapper.py [hunk 1]
```diff
@@ -261,8 +263,11 @@ def main():
     parser.add_argument(
         '--rosa-addons',
         type=str,
-        help='Comma separated list of addons to add to each cluster after installation is completed'
-    )
+        help='Comma separated list of addons to add to each cluster after installation is completed')
+    parser.add_argument(
+        '--rosa-flavour',
+        type=str,
+        help='AWS Flavor to use for infra nodes')
```
Comment (dry923) [inline rosa/rosa-wrapper.py:270]: This is for the worker nodes not infra

Comment (dry923) [reply rosa/rosa-wrapper.py:270]: disregard this. I had flavour and instnace-type swapped in my head. me--

**Review by dry923 — APPROVED** [review]
LGTM!

**Pull request merged**